pkg/term/winconsole/console_windows.go

```diff
@@ -211,14 +217,14 @@ func getStdHandle(stdhandle int) uintptr {
 	return uintptr(handle)
 }
 
-func StdStreams() (stdIn io.ReadCloser, stdOut io.Writer, stdErr io.Writer) {
+func WinConsoleStreams() (stdIn io.ReadCloser, stdOut, stdErr io.Writer) {
```

Comment:
In hindsight, I agree. It had been ill-named "StdStreams" which was wholly incorrect. ConsoleStreams would be better and less verbose. Good call.